students/HABTAMU/session03/mailroom.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin python3
+import sys
+
+donors_info = [("Bill Gates", [3500, 5500, 7500]),
+               ("Paul Alen", [3000, 3700, 3900]),
+               ("Jeff Benzo", [3300, 5000, 7500]),
+               ("Mark Zuckerberg", [33565.37, 465.37, 545.37, 7506]),
+               ("Warren Buffett", [3303.17, 334.17, 5080, 7500])
+               ]
+
+prompt = "\n".join(("\n Please choose from the following!:",
+                    " Choices are ?:",
+                    " T - To Send a Thank you!",
+                    " R - To Create a Report",
+                    " Q - To Exit",
+                    " >>> "))
+
+def send_thankyou():
```

Comment:
Might want to consider case sensitivity in this function.  Also, if user enters anything other than a number for a donation amount the program breaks.